### ISSUE TITLE
[REQUEST] Rewrite of Issue & PR templates

### ISSUE THREAD

**Issue by fabrice-v** [MobilityData/gtfs-validator#430]
**Is your feature request related to a problem? Please describe.**
Because we'll provide multiple environments and ways to run the validator either inside or outside Docker, we need to rewrite our issues template for better triaging.

**Describe the solution you'd like**
- checkmark about Docker environment
- have labels for criticality of issues (P0, P1, P2,... or 'critical' or 'crash')

**Describe alternatives you've considered**

**Additional context**
2. Issue templates/prioritization
- Cal-ITP concerned about issue prioritization/differentiation
- MobilityData acknowledges and is working on their triage process internally.
- MobilityData welcomes a PR with updated issue templates, etc.
E to Kay:
--> would you be interested to address issue you identified by updating the issue templates in a PR?
I'll take a look. 
To recap, the issue I raised is about explicitly flagging crash bugs, as they may hide other bugs that lurk behind them. It may be useful to have a Crash label.
